tools/pythonpkg/tests/fast/test_type.py

```diff
@@ -186,3 +187,19 @@ def test_attribute_accessor(self):
 
         child_type = type.v2.child
         assert str(child_type) == 'MAP(BLOB, BIT)'
+
+    def test_optional(self):
```

Comment:
Can you add a test with `Union[str, None]` or similar? Newer Python versions encourage `str | None` which is syntax sugar for that

Reply:
Thanks, added the test case 👍